docs/pds4_create_xml_index.rst

```diff
@@ -175,8 +186,9 @@ Label generation
   ``ancillary`` or ``Product_Metadata_Supplemental`` for ``supplemental``. Additional
   customization of the label can be provided with ``--label-user-input``.
 
-- ``--label-user-info``: Provide a file containing customization of the generated label.
-  The file must be in YAML format.
+- ``--label-user-input``: Provide a file containing customization of the generated label.
```

Comment:
The next big documentation sections to write are going to be on the format of this file and the config file, once we decide on whether they are two files or one. We should make this decision soon before people starting writing their own files and then need to convert.